src/NewsService.Mappers/Responses/NewsResponseMapper.cs

```diff
@@ -0,0 +1,104 @@
+﻿using LT.DigitalOffice.Broker.Requests;
+using LT.DigitalOffice.Broker.Responses;
+using LT.DigitalOffice.Kernel.Broker;
+using LT.DigitalOffice.Kernel.Exceptions.Models;
+using LT.DigitalOffice.NewsService.Mappers.ResponsesMappers.Interfaces;
+using LT.DigitalOffice.NewsService.Models.Db;
+using LT.DigitalOffice.NewsService.Models.Dto.Models;
+using LT.DigitalOffice.NewsService.Models.Dto.Responses;
+using MassTransit;
+using Microsoft.Extensions.Logging;
+using System;
+
+namespace LT.DigitalOffice.NewsService.Mappers.Responses
+{
+    public class NewsResponseMapper : INewsResponseMapper
+    {
+        private IRequestClient<IGetUserDataRequest> _userRequestClient;
+        private IRequestClient<IGetDepartmentRequest> _departmentRequestClient;
+        private readonly ILogger _logger;
+
+        private string GetUserFullName (Guid userId)
+        {
+            string fullName = null;
+
+            try
+            {
+                var request = IGetUserDataRequest.CreateObj(userId);
+                var response = _userRequestClient.GetResponse<IOperationResult<IGetUserDataResponse>>(request).Result;
+
+                if (!response.Message.IsSuccess)
+                {
+                    _logger.LogWarning($"Can't found user FIO. Reason: '{string.Join(',', response.Message.Errors)}'");
+                }
+
+                fullName = $"{response.Message.Body.LastName} {response.Message.Body.FirstName} {response.Message.Body.MiddleName}".Trim();
+            }
+            catch (Exception exception)
+            {
+                _logger.LogError(exception, "Exception on get user FIO data request.");
+            }
+
+            return fullName;
+        }
+
+        private string GetDepartmentName(Guid departmentId)
+        {
+            string name = null;
+
+            try
+            {
+                var request = IGetDepartmentRequest.CreateObj(departmentId);
+                var response = _departmentRequestClient.GetResponse<IOperationResult<IGetDepartmentResponse>>(request).Result;
+
+                if (!response.Message.IsSuccess)
+                {
+                    _logger.LogWarning($"Can't found department name. Reason: '{string.Join(',', response.Message.Errors)}'");
+                }
+
+                name = response.Message.Body.Name;
+            }
```

Comment:
use else for `name = response.Message.Body.Name`. If error was be  in Message, then name should be null

Reply:
done